### PR TITLE
Feature vs2015

### PR DESCRIPTION
**Pull request by B-C**
update to vs15
@jpcottin @jeanetteliu @mark-bittorrent @xercesblue please review

### CODE REVIEW

File: utp_internal.cpp [hunk 1]
```diff
@@ -1707,11 +1707,11 @@ void UTPSocket::apply_ccontrol(size_t bytes_acked, uint32 actual_delay, int64 mi
...
 			"delay_base:%u delay_sum:%d target_delay:%d acked_bytes:%u cur_window:%u "
-			"scaled_gain:%f rtt:%u rate:%u wnduser:%u rto:%u timeout:%d get_microseconds:"I64u" "
+			"scaled_gain:%f rtt:%u rate:%u wnduser:%u rto:%u timeout:%d get_microseconds:" I64u " "
 			"cur_window_packets:%u packet_size:%u their_delay_base:%u their_actual_delay:%u "
-			"average_delay:%d clock_drift:%d clock_drift_raw:%d delay_penalty:%d current_delay_sum:"I64u
-			"current_delay_samples:%d average_delay_base:%d last_maxed_out_window:"I64u" opt_sndbuf:%d "
-			"current_ms:"I64u"",
+			"average_delay:%d clock_drift:%d clock_drift_raw:%d delay_penalty:%d current_delay_sum:" I64u
+			"current_delay_samples:%d average_delay_base:%d last_maxed_out_window:" I64u " opt_sndbuf:%d "
+			"current_ms:" I64u "",
```
Comment (B-C) [inline utp_internal.cpp]: visual studio was interpreting I64u as a literal suffix

File: utp_types.h [hunk 2]
```diff
@@ -79,10 +79,6 @@
 	#define I64u "%Lu"
 #endif
 
-#ifdef WIN32
```
Comment (B-C) [inline utp_types.h]: it's defined now

**Comment by mark-bittorrent** [comment]
lgtm.

File: utp_types.h (hunk 2 above)
Comment (jpcottin) [inline utp_types.h]: where ?

Comment (B-C) [inline utp_types.h]: stdio.h

**Comment by xercesblue** [comment]
lgtm

**Pull request merged**